### PR TITLE
Introduce support for replicas

### PR DESCRIPTION
**Pull request by giacomomagni**

### CODE REVIEW

File: src/banana/benchmark/runner.py [hunk 1]
```diff
@@ -234,11 +231,11 @@ def load_external(self, session, t, o, pdf_name):
             db.Cache.external == self.external,
         )
         # if not found or multiple found, ext.one() will raise an Error
-        return pickle.loads(ext.one().result)
+        return pickle.loads(ext.one().ext_result)
```
Comment (AleCandido) [inline src/banana/benchmark/runner.py:234]: Also here: let's keep the name, `ext` is written at the beginning.

**Review by AleCandido — COMMENTED** [review]
@giacomomagni don't mind about fixing this, I'm taking care of `banana` improvements right now, I will complete this PR by myself :)

**Comment by giacomomagni** [comment]
yes sorry I'd like only to keep and possibly merge the stuff about the replicas, I'll revert the change on the database.

**Comment by AleCandido** [comment]
Don't mind, I will first merge #19, then I'll make sure everything is smooth about replicas and remove db :)

The plan is to merge both by today, if you want to help, take a moment to review #19.

**Comment by giacomomagni** [comment]
Okay, I will then make sure the script we used for the paper plots runs correctly.

**Comment by AleCandido** [comment]
I split the idea of keeping an internal cache in a separate issue #24 (and it will also be a separate table).

Nevertheless, this PR will be breaking for `yadmark` and `ekomark` (not a problem, luckily we have version constraints, so we can just make this a breaking change), but also for the current `navigator`: since the `cache` is changing (it will be a dictionary of caches) everything that uses that directly will break.
- [ ] check what is currently broken in `navigator`

In order to do this, most likely starting at least a small piece of #20 might be useful (since you need to produce some results, if they are self-contained in this repo it's easier to test, we don't have to bother with the version of the `ekomark`, nor to maintain two inconsisten versions, one using `banana` releases, and one using this branch).
- [ ] start `banana-mark` #20

Moreover, it would be nice to use the replicas to get more columns in comparisons: they should not show as an infinite number of columns in the `DFdict`s, but we can compute the standard deviation of the distribution (both for us and external).
- [ ] add replicas std everywhere it's possible to

In any case, this is breaking and not a priority, but whenever you have time @giacomomagni, it would be nice to have.

**Comment by felixhekhorn** [comment]
for sure in everyday life benchmarking we don't need replicas - so we need an option to opt-out

**Comment by giacomomagni** [comment]
I think this PR can be safely trashed now that we have `ekobox`, am I right?

**Comment by AleCandido** [comment]
Not sure how this couples to `ekobox`, since `ekobox` is not for benchmarking. But also `ekomark` is using this as a  "back-end" (underlying library).

However, if you there is no need for replicas in benchmarking that's fine :)

**Pull request closed without merging**